### ISSUE TITLE
Database backups should have skip-extended-insert option

### ISSUE THREAD

**Issue by ChemicalSailor** [ExpandedFronts/Revisr#79]
It should be possible to use skip-extended-insert option for mysqldump to make files more readable, and better for storing/diffing incremental DB changes. This may not be to everyone's taste, especially for large databases, so an option in settings view would be appropriate.

**Comment by mgshaw** [comment]
Good point. I'm thinking this will make it into 2.1- it will help with some improvements to batch processing that will be coming soon.
